packages/backend/src/auth/auth.service.ts

```diff
@@ -62,4 +68,89 @@ export class AuthService {
   async validateUser(payload: SignInPayload) {
     return await this.usersService.findByPayload(payload);
   }
-}
+
+  async resetPassword(email: string) {
+    try {
+
+      const user = await this.usersService.findByEmail(email);
+
+      if (!user) {
+        throw new HttpException(
+            ErrorMessage.USER_NOT_FOUND,
+            HttpStatus.NOT_FOUND,
+        );
+      }
+
+      const payload = { id: user.id, email: user.email };
```

Comment:
generowanie tokena to tez nie reset hasla do osobnej metody?

Reply:
poprawione